src/App.jsx

```diff
@@ -14,70 +15,79 @@ export const goods = [
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGood] = useState('Jam');
+  const [selected, setSelected] = useState(true);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function setSelector(item) {
+    if (item) {
+      setGood(item);
+      setSelected(true);
+    } else {
+      setGood('');
+      setSelected(false);
+    }
+  }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selected ? (
+          <>
+            {`${good} is selected`}
             <button
-              data-cy="RemoveButton"
+              data-cy="ClearButton"
+              onClick={() => setSelector()}
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+              className="delete ml-3"
+            />
+          </>
+        ) : (
+          'No goods selected'
+        )}
+      </h1>
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(item => (
+            <tr
+              data-cy="Good"
+              className={good === item ? 'has-background-success-light' : ''}
+              key={item}
             >
-              +
-            </button>
-          </td>
+              <td>
+                {good !== item ? (
+                  <>
+                    <button
+                      data-cy="AddButton"
+                      type="button"
+                      className="button"
+                      onClick={() => setSelector(item)}
+                    >
+                      +
+                    </button>
+                  </>
+                ) : (
+                  <>
+                    <button
+                      data-cy="RemoveButton"
+                      type="button"
+                      className="button is-info"
+                      onClick={() => setSelector()}
```

Comment:
```suggestion
                      onClick={setSelector}
```

Reply:
so an error occurs
![image](https://github.com/mate-academy/react_goods-selector-js/assets/55331267/c8ee9b7f-0b3c-4264-bdd2-d4077c9cb6e1)

Reply:
function setSelector expected one parameters, but got zero
you can execute this function with `null` parameter
onClick={() => setSelector(null)}